### DISCUSSION TITLE
Error when setting imageURI on a lineSymbol

### DISCUSSION THREAD

**Discussion by GabiN7** [pelicanmapping/osgearth#2147] [Support Q&A]
Hello, 

I have an error when I set the imageURI property on a lineSymbol, the error is :
`
[osgEarth]* [JavaScript] Compile error: SyntaxError: invalid escape (line 1)".
`

Graphically everything is going well, I can see the line with the desired texture.

Here is the code I used :

          osgEarth::Geometry* geom = new Polygon();

          geom->push_back( -160., -30. );
          geom->push_back(  150., -20. );
          geom->push_back(  160., -45. );
          geom->push_back( -150., -40. );

          Style geomStyle;
          Feature* feature = new Feature(geom, geoSRS);
          feature->geoInterp() = GEOINTERP_RHUMB_LINE;

          geomStyle.getOrCreate<LineSymbol>()->stroke()->color() = Color::Lime;
          geomStyle.getOrCreate<LineSymbol>()->stroke()->width() = 3000;
          geomStyle.getOrCreate<LineSymbol>()->stroke()->widthUnits() = Units::METERS;
          geomStyle.getOrCreate<LineSymbol>()->imageURI() = StringExpression("pathToMyTexture.png");
          geomStyle.getOrCreate<LineSymbol>()->tessellationSize()->set(75000, Units::METERS);
          
          geomStyle.getOrCreate<RenderSymbol>()->depthOffset()->enabled() = true;
          
          FeatureNode* gnode = new FeatureNode(feature, geomStyle);

Do you know what's the cause of this error ?

Thanks

**Comment by gwaldron** [comment]
It thinks your URI is Javascript. Instead try this:
```c++
imageURI()->setLiteral("pathToMyTexture.png");
```
